Mobile skeleton features

Draxeytina opened the pull request:
Add index.html
Add styles.css
Add project images

Draxeytina committed:
Add HTML and CSS linters

Draxeytina committed:
Add html file

added local html file

Draxeytina committed:
Add CSS file

Add local CSS stylesheet

Draxeytina committed:
Add CSS file

Draxeytina committed:
Add files via upload

Draxeytina committed:
Delete styles

Delete styles added in wrong folder

Draxeytina committed:
Update index.html

Update correct indentations

Draxeytina committed:
Update styles.css

Remove linter errors

Draxeytina committed:
Update styles.css

Remove linter errors

Draxeytina committed:
Update styles.css

Remove extra spaces

Draxeytina committed:
Update styles.css

Remove linters

Draxeytina committed:
Update styles.css

Draxeytina committed:
Update styles.css

Remove linter errors

Draxeytina committed:
Update styles.css

Updated hex letter case

Draxeytina committed:
Update index.html

Remove spaces
Update unclosed element

gtekle commented on index.html:
```diff
@@ -0,0 +1,45 @@
+
+</head>
+<body>
+    <div class="container">
```
- [x] You should use `<header>` tag for the header section for it has more semantic meaning instead of a generic `<div>` tag. Besides, would you please clarify why you need to wrap the single `<header>` with two layers of `<div>` tags? Please take a look at this [link](https://www.w3schools.com/html/html_layout.asp) for a basic HTML5 layout.

gtekle commented on index.html:
```diff
@@ -0,0 +1,45 @@
+            <nav class="navbar-header">
+                <ul>
+                    <li><p class="welcome-text">Welcome</p></li>
+                    <li><img class="burger" src="/images/Union.svg" alt=""></li>
```
- [x] The hamburger button is not displayed in your implementation.
Figma Design
![toolbar-design](https://user-images.githubusercontent.com/44057863/164773059-f509d524-a193-4ac7-b50c-7f445b8b4e18.png)
 
Your Implementation
![toolbar-stu](https://user-images.githubusercontent.com/44057863/164773086-3e86f213-6a60-4bba-9e3e-6df183ebddd1.png)

Draxeytina replied:
I really don't know why my images are failing to load. The path and file names are correct 😞

gtekle replied:
Hi @Draxeytina,
May you try adding a dot at the beginning of the path to indicate current directory? eg. `"./images/Union.svg"`

Draxeytina replied:
Thank you. I managed to add dots on the beginning of the file paths. I hope it is working now.

gtekle replied:
Yeah, the social media icons are now visible. Can you try the same thing with background image 
```
.about-container {
    background-image:
```

Draxeytina replied:
That's good news 😄 Thank you so much I'll never forget this tip! I did the same for the background image.

![image](https://user-images.githubusercontent.com/59999191/164799309-fe0e65c7-31e6-474e-9830-e849f1b7f22a.png)

gtekle replied:
:+1: Now, it is working correctly
![images-background](https://user-images.githubusercontent.com/44057863/164799337-ec99896a-0dab-42ce-aaba-77b54f282a70.png)
.

Draxeytina replied:
Amazing, thank you so much @gtekle

gtekle commented on index.html:
```diff
@@ -0,0 +1,45 @@
+        </header>
+    </div>
+    </div>
+    <div class="about-container">
```
- [x] You should use `<section>` tag as a container of the headline section for semantic reasons. Please refer link in the previous comment.
- [ ] You should add the background image for this section as in the Figma design. You can export the svg background image from the Figma design under `headline` with the name of `header-illustration-mobile`

Draxeytina replied:
Thank you! I managed to replace the <div> tags with <section> tags

Draxeytina replied:
For some reason my images are not loading for the Github repository. They are loading well on my local repository as shown on the screenshot. I'm still wondering why.

![image](https://user-images.githubusercontent.com/59999191/164793253-9a009e0a-15d8-4669-90b7-fd8ecb1867eb.png)

gtekle commented on index.html:
```diff
@@ -0,0 +1,45 @@
+        <div class="summary">
+            <section class="summary-section">
+                <h1 class="alignment about-heading1">Hey there. I’m Mariane</h1>
+                <h1 class="alignment about-heading2">I'm a software developer</h1>
```
- [ ] It is good that you have used `<h1>` for the `Hey there, I'm Mariane` text because it is the main heading you have in this html document. **However**, you need to have only one `<h1>` tag in HTML document for semantic reasons. Therefore, you should change the tag for this line `<h2>`

Draxeytina replied:
Noted, thank you.

Draxeytina committed:
Remove space on icon name

gtekle commented on index.html:
```diff
@@ -0,0 +1,45 @@
+            <div class="medium"><img src="/images/medium.svg" alt=""></div>
+            <div class="github"><img src="/images/github.svg" alt=""></div>
+            <div class="angellist"><img src="/images/angellist.svg" alt=""></div>
+        </div>
```
- [x] You should use `<ul> and  <li>` tags instead of of `<div>` for this are list of links.
- [x] You should also wrap the `<img>` tags with `<a>` so that they become clickable to take the user to your corresponding social media link. 
For example, 
```
<ul>
  <li>
    <a> 
         <img src="/images/twitter.svg" alt=""/>
     </a>
    </li>
    <li> ... </li>
  </ul>
```

- [x] The social media links are not displayed in your implementation. Make sure you are using the right icons and the correct path to the images.
Design
![figma-social-media-icons](https://user-images.githubusercontent.com/44057863/164776546-3f57c144-2297-4496-b24f-a8a74da31de5.png)

Implementation
![headline-bottom-secion-empty](https://user-images.githubusercontent.com/44057863/164776622-10712be8-c8ff-457f-bfd7-12a42c9fe05e.png)

Draxeytina replied:
Thank you I managed to replace the div tags with ul and li tags. 

![image](https://user-images.githubusercontent.com/59999191/164794109-b4e8ef71-3370-45e8-b0e2-884e903090d6.png)

gtekle reviewed (changes requested):
Hi @Draxeytina :wave:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
![almost-there-01](https://user-images.githubusercontent.com/44057863/164777006-7830c6c3-b460-4318-9add-5ee4c641581e.gif)


## Highlights
:heavy_check_mark: Good use of GitHub flow
:heavy_check_mark: No linter errors
:heavy_check_mark: Good organization of HTML code

## Required Changes ♻️
_Check the comments under the review._
- [x] You should add the update `Description` section, add `Getting Started` section, add `Author Name` to the Author section of your README as provided in the [template](https://github.com/microverseinc/readme-template/blob/master/README.md) by Microverse. 
- [x] You should stick the Figma design. (please see inline comments below)
- [x] You should use HTML5 semantic tags in instead of `<div>` or other generic tags. You may also refer this [article](https://www.w3schools.com/html/html5_semantic_elements.asp) on that.
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Draxeytina committed:
Update README.md

Update the description section

Draxeytina committed:
Update README.md

Add getting started section
Add author name

Draxeytina committed:
Update index.html

Remove div tags
Edit secondary heading to h2
Add header and section semantics
Add . to indicate current file directory

Draxeytina committed:
Update styles.css

Add . to indicate current folder for background image

SajjadAhmad14 reviewed (approved):
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Tinashe, 
## Congratulations 🎉 Thank you for implementing all the requirements 🙏
### Please go ahead and merge the branch!

## Happy Coding 👏👏👏
#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).
